nixos/modules/services/mail/postfix.nix

```
@@ -466,22 +466,18 @@ in
         ";
       };
 
-      sslCert = mkOption {
+      clientTlsCAFile = mkOption {
         type = types.str;
-        default = "";
-        description = "SSL certificate to use.";
+        default = "${pkgs.cacert}/etc/ssl/certs/ca-bundle.crt";
+        description = "File containing trusted certification authorities (CA) to verify certificates of mailservers contacted for mail delivery.
+        Defaults to NixOS trusted certification authorities.";
```

```suggestion
        description = ''
          File containing trusted certification authorities (CA) to verify certificates of mailservers contacted for mail delivery.
        '';
```
The default should be obvious enough.

---

How does the default render? If it's `(build of cacert-3.36)/etc/ssl/certs/ca-bundle.crt` or `/nix/store/732wsyyfia0l69nj5n1046gg35azjqnq-nss-cacert-3.63/etc/ssl/certs/ca-bundle.crt`, then `defaultText` might be in order.

---

Sure, `defaultText` is nicer.